clean.py

```diff
@@ -0,0 +1,436 @@
+# cspell:ignore dpkg, getent, lineinfile, nxapi, nxos, rglob, ruamel, sysvinit
+"""A test files updater."""
+
+import logging
```

Comment:
We do not need this file. This was a cleanup script used to lint all collections. Unrelated to CML work.